CMakeLists.txt

```diff
@@ -361,6 +363,7 @@ if (MSVC)
         /wd4800 # disable warning type' : forcing value to bool 'true' or 'false' (performance warning)
         /wd4503 # identifier' : decorated name length exceeded, name was truncated
         /wd4146 # unary minus operator applied to unsigned type, result still unsigned
+        /wd4244 # conversion from 'protobuf::int64' to 'int'
```

Comment:
where does such cast happen? do you also need to ignore this warning in other targets?

Reply:
This happens when I convert fields in functionproto in cpp2py.. I only notice the warning in onnx target..

Reply:
at least you would also need to ignore it in other platforms as well. maybe what's better is to find the place in pybind11 binding and fix it?